canvas.js

```diff
@@ -1996,13 +1996,17 @@ if (typeof Path2D !== 'function' ||
           function ratio(u, v) {
             return dot(u,v) / (mag(u)*mag(v))
           }
+
+          function clamp(value,min,max) {
```

Comment:
Super minor, but can you add spaces after the commas?

function clamp(value, min, max) {